chainer/backend.py

```diff
@@ -70,6 +71,15 @@ def copyto(dst, src):
             type(dst)))
 
 
+def _get_default_device_from_module(xp):
```

Comment:
This function is used in another file. How about making it public?

Reply:
Done.  I added comments because the function should not be used from a new interface.